scripts/start-console-k8s.sh

```diff
@@ -2,22 +2,28 @@
 
 set -euo pipefail
 
-# Set minikube IP and token (see scripts/configure-minikube.shcut)
-export BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT=${BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT:=https://$(minikube ip):8443}
+K8S_API_URL=$(kubectl config view -o jsonpath="{.clusters[?(@.name==\"$(kubectl config current-context)\")].cluster.server}")
+K8S_IP=$(echo ${K8S_API_URL} | cut -d: -f 2 | sed 's/[/]//g')
+K8S_PORT=$(echo ${K8S_API_URL} | cut -d: -f 3)
 
-# Default minkube home to user home, if MINIKUBE_HOME is not defined
-MINIKUBE_HOME_=${MINIKUBE_HOME:=$HOME}
-token=$(cat ${MINIKUBE_HOME_}/.minikube/files/etc/ca-certificates/token.csv 2>/dev/null | cut -f 1 -d,)
-export BRIDGE_K8S_AUTH_BEARER_TOKEN=${BRIDGE_K8S_AUTH_BEARER_TOKEN:=${token}}
+# Set IP and token (see scripts/configure-minikube/kind)
+export BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT=${BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT:=${K8S_API_URL}}
+
+# Default passwaed (see scripts/configure-minikube/kind)
+DEFAULT_TOKEN=abcdef.0123456789abcdef
+export BRIDGE_K8S_AUTH_BEARER_TOKEN=${BRIDGE_K8S_AUTH_BEARER_TOKEN:=${DEFAULT_TOKEN}}
+
+# Try to install forklift CRDs
+kubectl apply -f $(pwd)/deployment/hack/crds
```

Comment:
Can we move this line back to `configure-*` scritpts? 
1. The apply works for current cluster - it may not be the correct one. 
2. Not sure but it seems it can override manual changes (if any) done on the cluster

Reply:
moved back